charts/cmc-claim-store/requirements.yaml

```diff
@@ -3,11 +3,11 @@ dependencies:
     version: ~2.6.5
     repository: '@stable'
   - name: java
-    version: '>1.0.0'
+    version: ~1.0.7
     repository: '@hmcts'
     alias: claim-store
   - name: nodejs
-    version: 0.0.11
+    version: 0.1.4
     repository: '@hmcts'
   - name: rpe-feature-toggle-api
```

Comment:
this should be conditional based on a tag, see draft store or user-profile api

Reply:
yep, like it - have been meaning to use for postgres to use 1 instance :)